src/popup/components/annotationSummary/AnnotationSummary.tsx

```diff
@@ -5,7 +5,10 @@ import {
   AnnotationPPCategories,
   annotationPPCategoriesLabels,
 } from 'common/api/annotations';
-import _countBy from 'lodash/countBy';
+import _ from 'lodash';
```

Comment:
The idea was to not import whole lodash and increase the package size.
I just checked that, at least with Webpack 4, it does not matter. We can always simply import `_ from 'lodash'`